website_sale_adwords_conversion/models/website.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    google_adwards_conversoin_tracking = fields.Boolean()
+    google_adwords_conversion_key = fields.Char('Tracking ID')
+    google_adwords_event_key = fields.Char('Event ID')
+
+    @api.multi
+    def write(self, vals):
+        template_view = self.env.ref(
+            'website_sale_adwords_conversion.website_layout')
+        if template_view:
+            if 'google_adwards_conversoin_tracking' in vals:
+                template_view.active = \
+                    vals.get('google_adwards_conversoin_tracking')
+        return super().write(vals)
```

Comment:
Hmmm. I think the view should always be active.
Website views are tricky. You can have multiple websites, and each website has a copy of the view.

We shouldn't toggle this feature via activating/deactivating a view. The script element inside the view should be toggled via a `t-if="website.google_adwords_conversion_key"` instead